include/veriblock/serde.hpp

```diff
@@ -92,6 +113,52 @@ T readSingleBEValue(ReadStream& stream) {
       .readBE<T>();
 }
 
+/**
+ * Write single byte length value, which consists of
+ * `N bytes vector`
+ * Appends 1 byte data length to the stream
+ * @param stream write data to this stream
+ * @param value data that should be written
+ * @throws std::out_of_range if value size is too high
+ */
+void writeSingleByteLenValue(WriteStream& stream, const std::vector<uint8_t> value);
```

Comment:
pass vectors by reference
```suggestion
void writeSingleByteLenValue(WriteStream& stream, const std::vector<uint8_t>& value);
```

Reply:
well, since you only read this array before writing as bytes, you can replace it with `Slice<const uint8_t>`